backend/src/database/seeds/01_users.js

```diff
@@ -0,0 +1,31 @@
+
+exports.seed = function(knex) {
+  const tableName = "users";
+  // Deletes ALL existing entries
+  return knex(tableName).del()
+    .then(function () {
+      // Inserts seed entries
+      return knex(tableName).insert([
```

Comment:
yeah there should not be real user info for sure. i would say there are at least two classifications of seeds: 
1.  Foundation: basic setup of things that are always required to get the app going, no matter the installation. this would be things like: 
    - the base set of groups, roles, and permissions
    - the "root" user (if the app didn't have external auth)
    - dynamically configurable site options
 2. Mock: some objects to slingshot development and testing. this would be things like:
    - dummy users to have something to show and test with in the user list without having to create them each time
    - dummy contracts for the same reason, etc.
 
it saves a bunch of time of having to create those things manually for each person, and it also is consistent for debugging; someone else can run the same seeds to generate the same objects, then reproduce the bug with ease. otherwise you have to figure out how to get the same object into the database for someone else.

some seeders can even be programmatic; if you need dummy users, it just combines random names together and inserts them on the fly. sometimes its good to have them be random, so you get some fuzzing going on that reveals weird edge cases you wouldn't have thought of. sometimes you want them to be static though so they are consistent. 

from my experience of working with seeds before, i just treat them like CSVs (smart CSVs really) anyway. they are just datasets, so i don't mind if they get bigish. there should really only be enough to initialize and do some testing with though. if we want to make different seed sets for different cases, that is an option too. it really helps to formalize the data generation process and make it consistent/repeatable.

Reply:
my main point is what data you use there, ie fake data of fake people and fake things is more preferred over real.